app/models/application_record.rb

```diff
@@ -0,0 +1,8 @@
+# For backward compatibility with Redmine <= v5.x.
+# ApplicationRecord is inherited from ActiveRecord Models instead of ActiveRecord::Base in Redmine > 5.x
+# ref: https://www.redmine.org/issues/38975
+unless defined?(ApplicationRecord)
+  class ApplicationRecord < ActiveRecord::Base
+    self.abstract_class = true
+  end
```

Comment:
Can we use `ActiveRecord::Base` directly because Redmine < 6 still uses `ActiveRecord::Base` directly?

```suggestion
  ApplicationRecord = ActiveRecord::Base
```

Reply:
fix: [8d594e1](https://github.com/clear-code/redmine_full_text_search/pull/130/commits/8d594e1bf916820b5d6763cbb036e0e085f7b590)
Sure. Like you mentioned, we don't have to inherit `ActiveRecord::Base` using  `ApplicationRecord` with Redmine < 6.